docs/_static/worker_start.sh

```diff
@@ -9,7 +9,7 @@ function start_workers_for_tube {
     echo "Starting $2 workers for $1"
     for i in `seq 1 $2`
     do
-        teuthology-worker -v --archive-dir $ARCHIVE --tube $1 --log-dir $WORKER_LOGS &
+        teuthology-worker -v --archive-dir $ARCHIVE --tube $1 --log-dir $WORKER_LOGS > dev/null 2>&1 &
```

Comment:
Shouldn't we write it...somewhere? Rather than nowhere?

Reply:
yeah, ideally we should write it to somewhere. but

1. question `#1`:  what is "it". we have checked 3 different cases, namely:

   1. the `Hub.NOT_ERROR` case, where an exception is thrown when gevent stops or exits. they are bypassed.
   1. the `Hub.SYSTEM_ERROR` case, where the exception is pretty much only gevent is care understand/about. they are sent to https://github.com/gevent/gevent/blob/a8c1f1712bd2010cf28eb2f42774be273ce7dd4d/src/gevent/hub.py#L454 . i assume they will not be printed to stderr/stdout. but if they will, i guess it's something *unexpected*. and an i/o error helps us to notice these kinds of error in a way that not very "developer friendly".
   1. the other exceptions. they are printed to `log`.

2. question '#2': where is somewhere?  
as @batrick put in 2dbc296e98a18a1dc9e903944801b026bdb215ba, we already print the exception that we are interested in to the log file. so this is the best we can have for a  process running in background?

tl;dr: i think this is the best we can have.

Reply:
Okay, I must be missing how the exceptions can get printed in those cases, then?

Reply:
yeah, i think they are directed to `teuthology.log` with Patrick's change. =)

Reply:
@tchaikov 
a) You've missed 'root' slash for '/dev/null' device.
b) `somewhere`, for example, $WORKER_LOGS/worker-$1.$i.txt, often stderr contains some errors which does not get into python logger, and sometimes it is easier to find logs by constant filenames, in opposite to names based on random process id.